### PR TITLE
feat(pubsub): add ordering key samples

### PR DESCRIPTION
**Pull request by hongalex**

### CODE REVIEW

File: pubsub/topics/publish_ordering.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+			if err != nil {
+				// Error handling code can be added here.
+				fmt.Printf("Failed to publish: %s\n", err)
+				atomic.AddUint64(&totalErrors, 1)
+				return
+			}
+		}(result)
+	}
+
+	wg.Wait()
```
Comment (tbpg) [inline pubsub/topics/publish_ordering.go:91]: What is the part that needs to be done in the background here? `Publish` or `Get` or both? I'm wondering if we could split this into more than one loop to make it a little easier to follow.

File: pubsub/topics/publish_resume_ordering.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		{
+			message:     "message3",
+			orderingKey: "key1",
+		},
+		{
+			message:     "message4",
+			orderingKey: "key2",
+		},
+	}
+	for _, m := range messages {
```
Comment (tbpg) [inline pubsub/topics/publish_resume_ordering.go:70]: Same question here.

File: pubsub/topics/publish_ordering.go (hunk 1 above)
Comment (hongalex) [reply pubsub/topics/publish_ordering.go:91]: Yeah, all we want to show is how to publish with message ordering (lines 71-74). I'll try splitting this up until 2 loops.

**Pull request merged**